Updated cfg file for beampixel application for 2018 run

dinardo opened the pull request:
I changed from eras.Run2_2017 to eras.Run2_2018
and from PixelTracks_2017_cff to PixelTracks_cff
- Mauro.

Commit:
Updated cfg file for beampixel application for 2018 run

cmsbuild commented:
A new Pull Request was created by  @dinardo (Mauro) for CMSSW_10_0_X.

It involves the following packages:

DQM/Integration


@vazzolini, @kmaeshima, @dmitrijus, @cmsbuild, @jfernan2, @vanbesien can you please review it and eventually sign? Thanks.
@makortel, @threus, @batinkov, @thomreis this is something you requested to watch as well.
@davidlange6, @slava77, @fabiocos you are the release manager for this.

cms-bot commands are listed <a href="http://cms-sw.github.io/cms-bot-cmssw-cmds.html">here</a>

Commit:
Update beampixel_dqm_sourceclient-live_cfg.py

Updated tracking region for pixel tracks

cmsbuild commented:
Pull request #22783 was updated. @vazzolini, @kmaeshima, @dmitrijus, @cmsbuild, @jfernan2, @vanbesien can you please check and sign again.

Commit:
Update beampixel_dqm_sourceclient-live_cfg.py

cmsbuild commented:
Pull request #22783 was updated. @vazzolini, @kmaeshima, @dmitrijus, @cmsbuild, @jfernan2, @vanbesien can you please check and sign again.

Commit:
Update beampixel_dqm_sourceclient-live_cfg.py

Adjourned process.pixelTracksTrackingRegions.RegionPSet.originHalfLength to more approriate value.

cmsbuild commented:
Pull request #22783 was updated. @vazzolini, @kmaeshima, @dmitrijus, @cmsbuild, @jfernan2, @vanbesien can you please check and sign again.

dinardo commented:
Error
